backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/webservice/schema/RegionsQueryService.kt

```diff
@@ -1,28 +1,34 @@
 package app.ehrenamtskarte.backend.regions.webservice.schema
 
-import app.ehrenamtskarte.backend.common.webservice.schema.IdsParams
+import app.ehrenamtskarte.backend.common.webservice.DEFAULT_PROJECT
 import app.ehrenamtskarte.backend.regions.database.repos.RegionsRepository
-import app.ehrenamtskarte.backend.regions.webservice.dataloader.REGION_LOADER_NAME
 import app.ehrenamtskarte.backend.regions.webservice.schema.types.Region
 import com.expediagroup.graphql.annotations.GraphQLDescription
-import graphql.schema.DataFetchingEnvironment
 import org.jetbrains.exposed.sql.transactions.transaction
-import java.util.concurrent.CompletableFuture
 
 @Suppress("unused")
 class RegionsQueryService {
 
-    @GraphQLDescription("Return list of all regions.")
-    fun regions(): List<Region> = transaction {
-        RegionsRepository.findAll().map {
+    @GraphQLDescription("Return list of all regions in the given project.")
+    fun regionsInProject(project: String): List<Region> = transaction {
+        RegionsRepository.findAllInProject(project).map {
             Region(it.id.value, it.prefix, it.name, it.regionIdentifier)
         }
     }
 
-    @GraphQLDescription("Returns regions queried by ids.")
-    fun regionsById(
-        params: IdsParams,
-        environment: DataFetchingEnvironment
-    ): CompletableFuture<List<Region>> =
-        environment.getDataLoader<Int, Region>(REGION_LOADER_NAME).loadMany(params.ids)
+    @GraphQLDescription("Returns regions queried by ids in the given project.")
+    fun regionsByIdInProject(project: String, ids: List<Int>): List<Region?> = transaction {
```

Comment:
Changing the return type from `CompletableFuture<List<Region>>` to `List<Region?>` is not a breaking change, is it?

Reply:
You can check the generated schema I think.

Reply:
it does not :+1: